locator/token_metadata.cc

```diff
@@ -579,6 +579,7 @@ void token_metadata_impl::remove_endpoint(host_id endpoint) {
     _topology.remove_endpoint(endpoint);
     _leaving_endpoints.erase(endpoint);
     del_replacing_endpoint(endpoint);
+    remove_by_value(_replacing_endpoints, endpoint);
```

Comment:
On which code path this suppose to make a difference? Consistent topology change always re-create token metadata from the topology state, so if a node is no longer replacing some other node a new metadata is suppose to be created without this node.

Reply:
Okay, so let's close this PR then if it is not required anymore